stretchy-type/src/view.js

```diff
@@ -0,0 +1,13 @@
+import { adjustFontSize } from './utils';
+
+function handleFontSize() {
+	const elements = document.querySelectorAll('.wp-block-wpsp-stretchy-type');
+
+	elements.forEach(element => {
+		const content = element.innerHTML;
+		adjustFontSize( element, content );
+	});
+}
+
+document.addEventListener( 'DOMContentLoaded', handleFontSize );
+window.addEventListener('resize', handleFontSize );
```

Comment:
I tried this with the interactivity API, but it performed noticeably (visually) worse — the text would show up at the default size, and a split second later would resize — and it took a lot more code to do the same thing. 

https://github.com/a8cteam51/special-projects-blocks-monorepo/pull/5/commits/e4218c591a9012f480ebbb055f77af14452ac757

Maybe it could be optimized somehow.